internal/warehouse/githubActions.go

```diff
@@ -0,0 +1,418 @@
+package warehouse
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/google/go-github/v47/github"
+	"github.com/google/uuid"
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v4"
+	"github.com/mergestat/mergestat/internal/db"
+)
+
+func (w *warehouse) GithubActions(ctx context.Context, j *db.DequeueSyncJobRow) error {
```

Comment:
```suggestion
func (w *warehouse) GitHubActions(ctx context.Context, j *db.DequeueSyncJobRow) error {
```

a small nit - generally we try to capitalize the `H` in GitHub as that's how they ask to be spelled

Reply:
done